### ISSUE TITLE
List is not coded as a list (Andriod)

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#258]
Scenario
Samsung Galaxy J3 Prime 
Android version 7.0

With Talkback on, on the “Totally Anonymous” screen, some content (highlighted in the screenshot) visually appears to be a list but Talkback doesn’t read it as a list.

Expected result
Content should be read as a list.

WCAG
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Screenshots
![list](https://user-images.githubusercontent.com/12256067/85949566-8e447100-b925-11ea-9f0a-c3639cbe099b.gif)

**Comment by JuliannaR** [comment]
I am closing this issue as we will address all list reading issues together here https://github.com/cds-snc/covid-shield-mobile/issues/268
